doc-content/enterprise-only/installation/jms-broker-configure_proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='JMS-broker-configure_{context}']
+= Downloading and configuring AMQ Broker
+
+AMQ Broker enables your applications to communicate with any messaging provider. It specifies how components such as message-driven beans, Enterprise JavaBeans, and servlets can send or receive messages. 
+
+.Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product: AMQ Broker*
+* *Version: 7.2.0*
+. Click *Download* next to *Red Hat AMQ Broker 7.2.0* (`amq-broker-7.2.0-bin.zip`).
+. Extract the `amq-broker-7.2.0-bin.zip` file. 
+. Change directory to `amq-broker-7.2.0-bin/amq-broker-7.2.0/bin`.
+. Enter the following command and replace the following placeholders to create the broker and broker user:
++
+* `<HOST>` is the IP address or host name of the server where you installed AMQ Broker
+* `<AMQ_USER>` and `<AMQ_PASSWORD>` is a user name and password combination of your choice
```

Comment:
"are the user name and password of your choice"   Not great to say two things "is a _ and _ combination", imo. Bla and bla are vs Bla or bla is.

Also, end these bullets with periods, as you do correctly elsewhere. Unless you're doing so elsewhere just due to multi-sentences and don't like ending with a period when the subject is a replaceable. Seems fine to use periods to me.